modules/installation-gcp-permissions.adoc

```diff
@@ -29,6 +29,267 @@ account requires the following permissions. If you deploy your cluster into an e
 * Service Account User
 * Storage Admin
 
+The following are the required permissions for provisioning GCP infrastructure for creating and deleting the {product-title} cluster.
+
+.Required permissions for creating network resources
+[%collapsible]
+====
+* `compute.addresses.create`
+* `compute.addresses.createInternal`
+* `compute.addresses.get`
+* `compute.addresses.delete`
+* `compute.addresses.list`
+* `compute.addresses.use`
+* `compute.addresses.useInternal`
+* `compute.firewalls.create`
+* `compute.firewalls.get`
+* `compute.firewalls.delete`
+* `compute.firewalls.list`
+* `compute.forwardingRules.create`
+* `compute.forwardingRules.get`
+* `compute.forwardingRules.list`
+* `compute.forwardingRules.setLabels`
+* `compute.networks.updatePolicy`
+* `compute.networks.create`
+* `compute.networks.get`
+* `compute.networks.list`
+* `compute.routers.create`
+* `compute.routers.get`
+* `compute.routers.list`
+* `compute.routers.update`
+* `compute.routes.list`
+* `compute.subnetworks.create`
+* `compute.subnetworks.get`
+* `compute.subnetworks.list`
+* `compute.subnetworks.use`
+* `compute.subnetworks.useExternalIp`
+====
+
+.Required permissions for creating load balancer resources
+[%collapsible]
+====
+* `compute.regionBackendServices.create`
+* `compute.regionBackendServices.get`
+* `compute.regionBackendServices.list`
+* `compute.regionBackendServices.update`
+* `compute.regionBackendServices.use`
+* `compute.targetPools.addInstance`
+* `compute.targetPools.create`
+* `compute.targetPools.get`
+* `compute.targetPools.list`
+* `compute.targetPools.removeInstance`
+* `compute.targetPools.use`
+====
+
+.Required permissions for creating DNS resources
+[%collapsible]
+====
+* `dns.changes.get`
+* `dns.changes.create`
```

Comment:
Suggest to rearrange the permissions in ascending order as below
* `dns.changes.create`
* `dns.changes.get`

Reply:
Addressed